wallet/wallet-cli-commands/src/command_handler/mod.rs

```diff
@@ -1178,23 +1208,23 @@ where
                     .mint_tokens(selected_account, token_id, address, amount, self.config)
                     .await?;
 
-                Ok(Self::new_tx_submitted_command(new_tx))
+                Ok(Self::new_tx_command(new_tx, chain_config))
```

Comment:
Plz check all commands in this file that create a transaction that may or may not be broadcast, and ensure that the correct message is printed in the "config-broadcast no" mode.

E.g. as I can see, commands like IssueNewToken, IssueNewNft, CreateDelegation, DelegateStaking, WithdrawFromDelegation (probably some more) will currently report that the tx has been sent even if it hasn't.